ocs/defaults.py

```diff
@@ -6,3 +6,6 @@
 ROOK_CLUSTER_NAMESPACE = 'openshift-storage'
 KUBECONFIG_LOCATION = 'auth/kubeconfig'  # relative from cluster_dir
 CLUSTER_NAME = 'ocs-ci-cluster'
+k8s_api_version = 'storage.k8s.io/v1'
```

Comment:
Capital letters - same for all new additions

Reply:
DONE